packages/code-inline/package.json

```diff
@@ -39,7 +39,7 @@
   "devDependencies": {
     "@react-email/render": "workspace:*",
     "tsconfig": "workspace:*",
-    "tsup": "8.4.0",
+    "tsdown": "^0.15.1",
```

Comment:
can we use pinned dependencies here?

Reply:
also, why do we need to define `tsdown` as a `devDependency` in this `code-inline` package? All the other packages are not installing it directly